fixtures/csv_creation/digitization_example.json

```diff
@@ -4,32 +4,32 @@
     "bag_data": {
         "rights_statements": [
             {
-                "note": "Copyright term 120 years from date of creation.",
-                "end_date": "2023-12-31",
-                "start_date": "1961-01-01",
-                "jurisdiction": "us",
-                "rights_basis": "copyright",
+                "rights_basis": "other",
+                "start_date": "2019-01-01",
+                "end_date": null,
+                "basis_note": "These materials may only be accessed in a reading room setting.",
                 "rights_granted": [
-
+                    {
+                        "act": "publish",
+                        "grant_restriction": "disallow",
+                        "start_date": "1911-06-12",
+                        "end_date": null,
+                        "granted_note": ""
```

Comment:
It looks like this is a nullable field in Aquila so I think this would be `null` rather than an empty string.